### DISCUSSION TITLE
GHCR container packages not showing in the repo page

### DISCUSSION THREAD

**Discussion by patrickdung** [community/community#25770] [Actions]
<p>I used GH action to build and push to GHCR.</p>
<p>The package is shown in here:</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/users/patrickdung/packages/container/package/goatcounter-container">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/users/patrickdung/packages/container/package/goatcounter-container" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/38665827/181093186-ba5327fd-5496-422c-a0f9-0fae0d432771.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://github.com/users/patrickdung/packages/container/package/goatcounter-container" target="_blank" rel="noopener nofollow ugc">Build software better, together</a></h3>

  <p>GitHub is where people build software. More than 73 million people use GitHub to discover, fork, and contribute to over 200 million projects.</p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>But not in the repo’s package page:</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/patrickdung/goatcounter-container">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/patrickdung/goatcounter-container" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/38665827/181093187-7f85cd0e-170d-4619-8092-47d729dda71e.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/6/1/611692e7926a2cfe96cdb2fa304944b6b2cbbd41_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/6/1/611692e7926a2cfe96cdb2fa304944b6b2cbbd41_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/6/1/611692e7926a2cfe96cdb2fa304944b6b2cbbd41.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/1/611692e7926a2cfe96cdb2fa304944b6b2cbbd41_2_10x10.png"></div>

<h3><a href="https://github.com/patrickdung/goatcounter-container" target="_blank" rel="noopener nofollow ugc">GitHub - patrickdung/goatcounter-container: GH actions to build unofficial...</a></h3>

  <p>GH actions to build unofficial Docker images for GoatCounter - GitHub - patrickdung/goatcounter-container: GH actions to build unofficial Docker images for GoatCounter</p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
<p>
Packages:<br>
No packages published.</p>
<p>The visibility of the package is set to public.</p>
<p>Thanks!</p>

**Comment by lindluni** [comment, marked as answer]
<p>Hi <a class="mention" href="/patrickdung">@patrickdung</a></p>
<p>You need to associate your container with your repository, you can do this by adding the required  LABEL in the Dockerfile, or by manually associating the container in the package settings.</p>
<p>You can read the documentation here on how to associate your container with a repository: <a href="https://docs.github.com/en/packages/learn-github-packages/connecting-a-repository-to-a-package" class="inline-onebox" rel="noopener nofollow ugc">Connecting a repository to a package - GitHub Docs</a></p>

**Comment by patrickdung** [comment]
<p>Hi <a class="mention" href="/lindluni">@lindluni</a><br>
It’s working now! Thanks for the help.</p>
